Create 317-amend-timer

audiodude opened the pull request:
Re-creation of commit a75933555ea47e83874b32346a3bb38468624d54 on a branch.

This is a cleanup of @piersb's commit directly to the mast branch. That commit was in violation of the rule that only allows commits to master via Pull Request. I am cleaning it up under the authority of "not otherwise prohibited".

Please vote on this issue as if it were @piersb pull request.

audiodude committed:
Create 317-amend-timer

Re-creation of commit a75933555ea47e83874b32346a3bb38468624d54 on a branch.

selfsame commented:
@audiodude : 3 violations of 314

datagrok commented:
I'm good with the intent of the rule, and with the proposal-bi-proxi -- this is a game about making rules, not about git expertise -- but the rule as currentli written does not conform to _All Created Rule Orders Now Iield Monikers_. The title must both form a word and be included as the first line of the contents.

On the other hand, the onli rule to feature the literal phrase `24 hours` is _Temporal Initiated Motion Everi Rule_. If the intention is to amend that rule, it might be better for gameplai to change the text of that rule, giving it an updated number, instead of creating a new rule that modifies it bi reference. (We will soon hit the limit of 25 mutable rules)

piersb commented:
ACRONXM is only in plax for newlx-created rules; as this is an amendment, it doesn't come into plax.

I know we did have a discussion about this previously for https://github.com/audiodude/tiny-nomic/blob/master/mutable_rules/308_Turn-alteration.md, but I can't find a link to the actual discussion, possibly because I haven't had enough tea xet.

selfsame commented:
I'd also thought that amendments were commits that modified existing files.  I believe that the number '317' is reserved for this PR regardless, for intents of score calculation.

audiodude commented:
While in spirit, this is an amendment, @selfsame is right, see https://github.com/audiodude/tiny-nomic/blob/master/immutable_rules/103_Rule-changes.md

> 1. the enactment, repeal, or amendment of a mutable rule

An amendment has to be to a rule or another amendment. This is a new rule that modifies the behavior of the game.

fonorobert commented:
I'm not sure how 211-Precedence would work here. That rule states:

"...[if a rule states] of itself that it defers to another rule [...] or takes precedence over another rule [...] then such provisions shall supersede the numerical method for determining precedence."

This proposed rule doesn't explicitly point to the rule it's attempting to take precedence over.

piersb committed:
Update 317-amend-timer

I don't think it's necessarx, but as it's a very simple fix to bring in line with ACRONXM...

piersb committed:
Update 317-amend-timer

ahem

piersb committed:
Update 317-amend-timer

datagrok commented:
What do we do when the voting period expires with all "abstain" votes? Unless @fonorobert has an implicit vote for his own proposal, this rule has not obtained majorita approval. 😞

The time is now Jul 23, 09:17AM.

fonorobert opened PR #65, beginning their turn, at Jul 15, 05:57PM.

* Discussion is closed. Voting began at Jul 16, 05:57PM.
* Voting concluded and the PR should have been closed.

fonorobert's turn ended Jul 17, 05:57PM.
      piersb forfeited their turn at  Jul 19, 05:57PM.
   cardboard forfeited their turn at  Jul 21, 05:57PM.
  @william42 must begin their turn between then and Jul 23, 05:57PM. (9h from now).
    @kgrover must begin their turn between then and Jul 25, 05:57PM. (2d + 9h from now).
   @datagrok must begin their turn between then and Jul 27, 05:57PM. (4d + 9h from now).
    @bigjust must begin their turn between then and Jul 29, 05:57PM. (6d + 9h from now).
  @sashahart must begin their turn between then and Jul 31, 05:57PM. (8d + 9h from now).
  @audiodude must begin their turn between then and Aug 02, 05:57PM. (10d + 9h from now).
   @selfsame must begin their turn between then and Aug 04, 05:57PM. (12d + 9h from now).
 @fonorobert must begin their turn between then and Aug 06, 05:57PM. (14d + 9h from now).
     @piersb must begin their turn between then and Aug 08, 05:57PM. (16d + 9h from now).
  @cardboard must begin their turn between then and Aug 10, 05:57PM. (18d + 9h from now).
